### PR TITLE
Update extension gallery files for September release

### PR DESCRIPTION
**Pull request by kburtram**

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -2556,6 +2556,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "54",
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json]: do we only want this in insiders?

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -2556,6 +2556,63 @@
...
+					},
+					"versions": [
+						{
+							"version": "0.0.3",
+							"lastUpdated": "08/14/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json]: Isn't this going to mean users have to go to the page - download it and then install it manually? Is that our SOP for non-first party extensions? It'd be easier to just have this be a VSIX package link and then point at https://github.com/alx-ppv/ADS-delete-database/releases/download/0.0.3/deletedb-0.0.3.vsix if that's not the case then right?

File: extensionsGallery-insider.json [hunk 3]
```diff
@@ -2564,11 +2621,11 @@
 					"metadataItems": [
 						{
 							"name": "TotalCount",
-							"count": 42
+							"count": 44
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json]: Was this not updated correctly last time?

File: extensionsGallery-insider.json (hunk 2 above)
Comment (kburtram) [reply extensionsGallery-insider.json]: Yes, currently 3rd-party extensions need to be manually side-loaded.  Once we have a proper marketplace we plan to improve this experience is so all extensions can be installed directly within the product.

File: extensionsGallery-insider.json (hunk 3 above)
Comment (kburtram) [reply extensionsGallery-insider.json]: yes, most the time people don't update this.  We don't use it for anything, so perhaps we should remove it, but in the meantime I try to update it to the correct value whenever editing these files.

**Pull request merged**